client/package.json

```diff
@@ -13,15 +13,17 @@
   "dependencies": {
     "@tanstack/react-query": "^4.36.1",
     "@tanstack/react-query-devtools": "^4.36.1",
-    "axios": "^1.6.8",
+    "axios": "^1.7.7",
```

Comment:
Updating the project library versions along with the feature changes isn't ideal. Please revert the version updates so I can address them in a separate PR.